crates/subspace-networking/src/behavior/persistent_parameters.rs

```diff
@@ -163,9 +170,29 @@ impl NetworkingParametersManager {
             known_peers: cache,
             networking_parameters_save_delay: Self::default_delay(),
             bootstrap_addresses,
+            collection_batcher: CollectionBatcher::new(
+                NonZeroUsize::new(PEERS_ADDRESSES_BATCH_SIZE)
+                    .expect("Manual non-zero initialization failed."),
+            ),
         })
     }
 
+    // Returns known addresses from networking parameters DB. It removes p2p-protocol suffix.
```

Comment:
It doesn't remove anything and I'm not sure that is even helpful in the API docs, it is kind of obvious from the signature.